Rename prop.TypeUrl -> prop.TypeURL

Linters report:

```
prop/prop.go:33:2: const TypeUrl should be TypeURL (golint)
	TypeUrl           InputType = "url"
```

This is similar to conclusions in https://github.com/gopherjs/vecty/issues/24, we should rename this and perhaps vet others in this package.

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -17,9 +17,6 @@ issues:
 
   # List of regexps of issue texts to exclude, empty list by default.
   exclude:
-    # https://github.com/gopherjs/vecty/issues/224
-    - "const TypeUrl should be TypeURL"
-
     # https://github.com/gopherjs/vecty/issues/226
     - "comment on exported function Timeout should be of the form"
 

--- a/prop/prop.go
+++ b/prop/prop.go
@@ -30,7 +30,7 @@ const (
 	TypeTel           InputType = "tel"
 	TypeText          InputType = "text"
 	TypeTime          InputType = "time"
-	TypeUrl           InputType = "url"
+	TypeURL           InputType = "url"
 	TypeWeek          InputType = "week"
 )
 

